tensorflow_recommenders_addons/dynamic_embedding/core/utils/utils.h

```diff
@@ -42,4 +42,24 @@ This code is for compatibility.*/
 }  // namespace recommenders_addons
 }  // namespace tensorflow
 
+// For propagating errors when calling a function but not return status.
+#if TF_VERSION_INTEGER >= 2130
+#define LOG_IF_ERROR(...)                  \
```

Comment:
Better name it `TFRA_LOG_IF_ERROR` because the macro has no name scope.

Reply:
fixed